features/fixtures/ios-swift-cocoapods/iOSTestApp/AppDelegate.swift

```diff
@@ -15,6 +15,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let arguments = ProcessInfo.processInfo.arguments
         var delay: TimeInterval = 0
         var eventType = "none"
+        var eventMode = "regular"
```

Comment:
What's the difference between an event type and an event mode? Do they need to be separate concepts?

Reply:
There needs to be separation between “run this scenario’s crashing code” and “configure Bugsnag for this scenario (and not run the crashy code”. Without this, an individual scenario can’t configure Bugsnag for relaunching the app.